src/frontend/packages/lib_markdown/src/components/ScreenDispositionSelector/index.spec.tsx

```diff
@@ -9,64 +8,34 @@ describe('<ScreenDispositionSelector />', () => {
   it('send disposition changes', () => {
     const setScreenDisposition = jest.fn();
 
-    // split -> editor only
+    // editor -> rendering
     const { rerender } = render(
       <ScreenDispositionSelector
-        screenDisposition={ScreenDisposition.splitScreen}
+        screenDisposition={ScreenDisposition.editor}
         setScreenDisposition={setScreenDisposition}
       />,
     );
 
-    userEvent.click(screen.getByTestId('disposition-editor-only'));
+    screen.getByRole('tab', { name: 'Preview' }).click();
```

Comment:
(Not a review) I was wondering if there is a difference by doing like that instead of using userEvent.